module/Makefile

```diff
@@ -48,10 +48,17 @@ clean:
 	rm -rf *.o *.ko *.symvers *.mod.c .*.cmd Module.markers modules.order *.o.*
 	rm -rf .tmp_versions .rapiddisk.o.d *.unsigned *.sdtinfo.c .ctf/ .cache.mk
 
-.PHONY: dkms
-dkms:
-	sudo mkdir -pv /usr/src/rapiddisk-$(VERSION)/
-	sudo cp -v * /usr/src/rapiddisk-$(VERSION)/
-	sudo dkms add -m rapiddisk -v $(VERSION)
-	sudo dkms build -m rapiddisk -v $(VERSION)
-	sudo dkms install -m rapiddisk -v $(VERSION)
+dkms-install:
+ifeq ($(shell dkms status rapiddisk/$(VERSION) -k $(KVER) | grep '$(KVER)' >/dev/null && echo 0 || echo 1 ),0)
+	$(error rapiddisk version $(VERSION) is already installed for kernel $(KVER). Use 'make dkms-remove' first.)
```

Comment:
please change the dkms-remove to dkms-uninstall

Reply:
Seems like this old 'dkms-remove' was copied back. As soon as you re-add that I will approve/merge. I tested the rest of the code and all works/looks great. Thank you!

Reply:
> Seems like this old 'dkms-remove' was copied back.

Too hot these days down here....

>As soon as you re-add that I will approve/merge. I tested the rest of the code and all works/looks great. 

It should be ok... I hope! 

> Thank you!

It was a pleasure. :)

Regards